php/utils.php

```diff
@@ -506,9 +506,11 @@ function parse_url( $url ) {
 
 /**
  * Check if we're running in a Windows environment (cmd.exe).
+ *
+ * $param mixed $test Testing only.
  */
-function is_windows() {
-	return strtoupper(substr(PHP_OS, 0, 3)) === 'WIN';
+function is_windows( $test = null ) {
```

Comment:
I'd prefer an environment variable over a test argument.

An environment variable makes it easier for an end user to never accidentally pass an argument.